lib/iris/fileformats/grib/_load_convert.py

```diff
@@ -442,6 +445,14 @@ def ellipsoid(shapeOfTheEarth, major, minor, radius):
             major *= 1000
             minor *= 1000
         result = icoord_systems.GeogCS(major, minor)
+    if shapeOfTheEarth == 4:
+        # Earth assumed oblate spheroid as defined in IAG-GRS80 model.
+        result = icoord_systems.GeogCS(6378137,
+                                       inverse_flattening=298.257222101)
+    if shapeOfTheEarth == 5:
+        # Earth assumed represented by WGS84 (as used by ICAO since 1998).
+        result = icoord_systems.GeogCS(6378137,
+                                       inverse_flattening=298.257223563)
```

Comment:
@lbdreyer I checked the grib docs and couldn't see the magic numbers, but they were buried in the `GribWrapper` ... which is good enough for me, thanks.